city-infrastructure-platform/utils.py

```diff
@@ -0,0 +1,31 @@
+import os
```

Comment:
We're working with an open source project that is borrowing code from Numpy, another open source project, so we definitely should include the licence Numpy is using at the start of this file.

Reply:
Good point! :+1: